templates/patient/partials/portal.html.twig

```diff
@@ -28,20 +28,27 @@ The Patient Portal card for the Medical Record Dashboard
             <p class="font-weight-bold">{{  "Portal Status"|xlt }}</p>
             <p> {{ "Patient Portal Site Address Not Set Properly. Admin Can Correct in Config."|xlt }} </p>
         </div>
-    {% endif %}    
-
+    {% endif %}
 
     {% if isPortalAllowed == false %}
         <div class="alert alert-warning" role="alert">
             <p class="font-weight-bold">{{  "Portal Access"|xlt }}</p>
             <p> {{ "Allow Patient Portal in Choices."|xlt }} </p>
-        </div> 
+        </div>
+    {% elseif isContactEmail == false %}
```

Comment:
what happens if there is no contact email but not enforcing email for sign on?
(looks like nothing will show up)

Reply:
prob do:
```
{% elseif isContactEmail == false  and isEnforceSigninEmailPortal == true %}
```